nixos/modules/config/console.nix

```
@@ -148,10 +148,9 @@ in
           ''}
         '');
 
-        boot.initrd.systemd.contents = {
-          "/etc/kbd".source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share";
-          "/etc/vconsole.conf".source = vconsoleConf;
-        };
+        boot.initrd.systemd.contents."/etc/vconsole.conf".source = vconsoleConf;
+        boot.initrd.systemd.contents."/etc/kbd" = lib.mkIf cfg.earlySetup { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share"; };
+        boot.initrd.systemd.contents."/etc/kbd/keymaps" = lib.mkIf (!cfg.earlySetup) { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share/keymaps"; };
```

IMO this was more readable as:
```suggestion
        boot.initrd.systemd.contents = {
          "/etc/vconsole.conf".source = vconsoleConf;
          "/etc/kbd" = lib.mkIf cfg.earlySetup { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share"; };
          "/etc/kbd/keymaps" = lib.mkIf (!cfg.earlySetup) { source = "${consoleEnv config.boot.initrd.systemd.package.kbd}/share/keymaps"; };
        };

```

---

Like this? Added a comment as well